### PR TITLE
Sync up VAOS Cerner detection with static content

### PR DESCRIPTION
**Pull request by jbalboni**
## Description
This updates puts VAOS in sync with the static content checks for Cerner sites, and fixes some VAOS bugs due to the FHIR conversion.

## Testing done
Local and unit testing

## Acceptance criteria
- [ ] Cerner sites are detected correctly

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/platform/user/selectors.js [hunk 1]
```diff
@@ -11,9 +12,13 @@ export const isLOA1 = state => selectProfile(state).loa.current === 1;
 export const isMultifactorEnabled = state => selectProfile(state).multifactor;
 export const selectAvailableServices = state => selectProfile(state)?.services;
 export const selectPatientFacilities = state =>
-  selectProfile(state)?.facilities?.filter(
-    f => !f.facilityId.startsWith('742'),
-  ) || null;
+  selectProfile(state)?.facilities?.map(({ facilityId, isCerner }) => ({
```
Comment (jbalboni) [inline src/platform/user/selectors.js:15]: This is the main global change. This updates the `isCerner` flag to reflect either the MPI status flag or the hardcoded list in the utilities/cerner folder (if the isCernerLive flag is flipped). This should allow testing in staging and still allow the website team to control when new content goes live and the VAOS team to be in sync with that or automatically pick up new Cerner sites.

File: src/platform/utilities/cerner/index.js [hunk 2]
```diff
@@ -2,6 +2,7 @@
 import environment from 'platform/utilities/environment';
 
 export const CERNER_FACILITY_IDS = ['668', '757'];
+export const isCernerLive = !environment.isProduction();
```
Comment (jbalboni) [inline src/platform/utilities/cerner/index.js:5]: This probably makes sense as a feature flag, but I think that's complicated with the website content work.

File: src/applications/vaos/actions/appointments.js [hunk 3]
```diff
@@ -11,18 +11,11 @@ import { resetDataLayer } from '../utils/events';
...
   updateAppointment,
   updateRequest,
 } from '../api';
 
-import {
-  getOrganizations,
-  getRootOrganization,
-  getSiteIdFromOrganization,
-} from '../services/organization';
-
```
Comment (narin) [inline src/applications/vaos/actions/appointments.js:25]: My fault.   Should have removed these after moving express care into it's own actions file.

**Review by narin — APPROVED** [review]
This looks good to me overall. I see that a good amount of these changes involves removing the code we had in there to remove `var` from facility IDs.  Could use a little more clarity on how we are handling these moving forward.  Is it because we now know what the IDs will be looking like coming from VSP?

**Comment by jbalboni** [comment]
@narin I don't think there's any change to what we've been doing overall, the code in here was mostly to make the the `getVARFacilityId` return a VistA facility id without `var`, since most uses of that function were later removing the `var` part anyway.

**Review by bkjohnson — APPROVED** [review]

**Review by ncksllvn — APPROVED** [review]
LGTM

**Pull request merged**